modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,11 +83,74 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Installing Istio
 
-You can install Istio 1.4.x or 1.5.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x, 1.5.x, or 1.4.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+
+=== Install Istio 1.6.x
+
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
+
+. Download Istio using the https://istio.io/docs/setup/[Istio documentation].
+. Install Istio with the following configurations:
```

Comment:
How does one "install with configurations"? Doesn't one first install and then configure?
1. Download
2. Install
3. Enable flag
4. Enable telemetry
5. Install manifest? What is a manifest? It sounds like it is separate from Istio, which has already been installed. What is a full manifest vs. a not full manifest?

The topic titles here are incorrect if the task includes three tasks:
1. Download
2. Install
3. Configure

Review to ensure that the language used throughout this topic is precise.

Reply:
Done! Thanks!